src/components/ActionRow/stylesheet.scss

```diff
@@ -26,7 +27,66 @@
       }
 
       &:last-child {
+        .icon {
+          padding-right: 16px;
+        }
+      }
+
+      .icon {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        font-size: 0.8em;
+        padding: 8px;
+        background-color: $color-dark;
+
+        svg {
+          opacity: $opacity-inactive;
+        }
+      }
+
+      &:hover {
+        .icon {
+          svg {
+            opacity: 1;
+          }
+        }
+      }
+    }
+  }
+
+  &:hover .actions {
+    visibility: visible;
+  }
+}
+
+.App.dark .ActionRow {
```

Comment:
Most of the properties of `.App.dark .ActionRow` are the duplicates of `.App.light .ActionRow`.

`.ActionRow` should have common properties that don't depend on themes, and `.App.light .ActionRow` and `.App.dark .ActionRow` should only contain theme-specific properties.

Reply:
It goes the same for any other stylesheets.